### PR TITLE
Update Font Awesome dependencies

### LINKED ISSUE #9101: Unable to build locally
I am unable to build on my local machine using `npm run all` command, which throws an error related to not being able to find icon in font-awesome. It was working fine a few hours back ( if that helps ).

### URL

_No response_

### How to reproduce the issue?

Run `npm run all` command and you'll see the following error:

```
zeeshan@Zeeshans-MacBook-Air iD % npm run all

> iD@2.21.0-dev all
> run-s clean build dist


> iD@2.21.0-dev clean
> shx rm -f dist/esbuild.json dist/*.js dist/*.map dist/*.css dist/img/*.svg


> iD@2.21.0-dev build
> run-s build:css build:data build:js


> iD@2.21.0-dev build:css
> node scripts/build_css.js


🏗   Building css...
👍  css built: 134.63ms


> iD@2.21.0-dev build:data
> shx mkdir -p dist/data && node scripts/build_data.js


🏗   Building data...
Error: No FontAwesome icon for fas-th-list
TypeError: Cannot read properties of undefined (reading 'html')
    at /Users/zeeshan/work/iD/scripts/build_data.js:229:75
    at Array.forEach (<anonymous>)
    at writeFaIcons (/Users/zeeshan/work/iD/scripts/build_data.js:224:23)
    at /Users/zeeshan/work/iD/scripts/build_data.js:114:7
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 9)

ERROR: "build:data" exited with 1.
ERROR: "build" exited with 1.
```

### Screenshot(s) or anything else?

Currently tested on `develop` branch

### Which iD Editor versions do you see the issue on?

_No response_

### Which browsers are you seeing this problem on?

_No response_

### PR DESCRIPTION
**Pull request by arch0345**
Fixes issue where aliases of [renamed Font Awesome icons](https://fontawesome.com/docs/web/setup/upgrade/whats-changed#icons-renamed-in-version-6) weren't recognized when running `npm run build`

Closes #9101, closes #9048, closes #9049, closes #9050, closes #9051

### CODE REVIEW

**Review by 1ec5 — COMMENTED** [review]
This is working well for me locally. What’s left before this PR is ready to move out of Draft status?

**Comment by arch0345** [comment]
> What’s left before this PR is ready to move out of Draft status?

Just wanted to make sure this didn't break anything else. After looking through the [scripts](https://github.com/openstreetmap/iD/tree/develop/scripts), it looks like the build script is the only one that uses these dependencies.

File: svg/fontawesome/fas-child.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="child" class="svg-inline--fa fa-child" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path fill="currentColor" d="M224 144c39.75 0 72-32.25 72-72S263.8-.0004 224-.0004S151.1 32.25 151.1 72S184.3 144 224 144zM415.1 110.8c-13.89-17.14-39.06-19.8-56.25-5.906L307.6 146.4c-47.16 38.19-120.1 38.19-167.3 0L89.17 104.9C72.02 91 46.8 93.67 32.92 110.8C19.02 128 21.66 153.2 38.83 167.1l51.19 41.47c11.73 9.496 24.63 17.16 37.98 23.92L127.1 480c0 17.62 14.38 32 32 32h15.1c17.62 0 32-14.38 32-32v-112h32V480c0 17.62 14.38 32 32 32h15.1c17.62 0 32-14.38 32-32l-.0001-247.5c13.35-6.756 26.25-14.42 37.97-23.91l51.2-41.47C426.3 153.2 428.1 128 415.1 110.8z"></path></svg>
```
Comment (1ec5) [inline svg/fontawesome/fas-child.svg:1]: I kinda liked the older icon, but we can live with the new design if it means builds will work again.

**Review by 1ec5 — APPROVED** [review]
Thanks for fixing this build issue!

**Pull request merged**